### ISSUE TITLE
feat: jvp for jacfwd with vectors

### ISSUE THREAD

**Issue by nstarman** [GalacticDynamics/coordinax#293]
This would speed up and simplify the velocity / acceleration representation transformations.
https://github.com/GalacticDynamics/coordinax/blob/fe0bfbae7c34b0bc71b05d804d00fa472bf4bb98/src/coordinax/_src/vectors/transform/velocities.py

**Comment by nstarman** [comment]
Re-checked on current `main`. **The premise no longer holds, and the stated need appears met.**

The file this issue points at — `src/coordinax/_src/vectors/transform/velocities.py` — no longer exists; neither does `_src/vectors/`. The vector layer was restructured into `Point`/`Tangent`/`Coordinate` over charts.

Forward-mode AD through a chart conversion works and is correct:

```python
def f(x):
    return u.ustrip("kpc", cx.Point.from_(x, "kpc").cconvert(cx.sph3d).data["r"])

jax.jacfwd(f)(jnp.array([1.0, 2.0, 3.0]))
#  [0.26726127, 0.53452253, 0.8017837]
```

which is exactly `(x, y, z) / |r|` for `|r| = sqrt(14) = 3.7417` — the analytic gradient of the radius.

What I have **not** established is the performance half. This issue asked for a custom `jvp` rule to *speed up* the transformations, and I have not benchmarked whether a hand-written rule would still beat what `jacfwd` produces now. So: correctness is no longer a reason to keep this open; a measured speed case would be. Without one, I would close it and reopen against a benchmark if the need resurfaces.
